lib/nghttp2_session.c

```diff
@@ -3163,7 +3163,7 @@ ssize_t nghttp2_session_mem_send(nghttp2_session *session,
 }
 
 int nghttp2_session_send(nghttp2_session *session) {
-  const uint8_t *data;
+  const uint8_t *data = NULL;
```

Comment:
Probably, this is not necessary since we always end up to call nghttp2_session_mem_send_internal, and we assign NULL to data.